options/test_util.go

```diff
@@ -0,0 +1,34 @@
+package options
+
+import (
+	"context"
+	"errors"
+
+	"github.com/mongodb/grip/message"
+	"github.com/mongodb/grip/send"
+)
+
+// MockSender is a simple mock implementation of the grip/send.Sender
```

Comment:
Can we just put this in grip and make the mock configurable?

Reply:
Yeah I was thinking we could do that, but this would be too simple for a generalized mock sender. I can implement one in a separate ticket.

Reply:
https://jira.mongodb.org/browse/EVG-13443